src/librustc/middle/cstore.rs

```diff
@@ -122,7 +122,6 @@ pub struct ExternCrate {
 /// can be accessed.
 pub trait CrateStore<'tcx> {
     // item info
-    fn describe_def(&self, def: DefId) -> Option<Def>;
```

Comment:
Whoa, how is this even possible?! Even if you managed to remove uses of it, this is important for other things.
E.g. I was telling @solson about `describe_def` being usable for telling apart `static` and `static mut`.
And I've used it in an upcoming PR to replace `is_extern_item`.